.github/CODEOWNERS

```diff
@@ -9,6 +9,62 @@
 # explicitly taken by someone else.
 *                               @GoogleCloudPlatform/python-samples-owners
 
+<<<<<<< HEAD
```

Comment:
Merge conflict? 😕

Reply:
Argh, I don't see where the conflict is. We'll have @hujessica remove this from the PR and then it can be updated separately.

Reply:
(This is fixed now. 🙂  )